src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+    view_all_sponsors: スポンサーの皆様を見る
+
+  donations:
+    title: 最近の寄付
+    view_all_donations: 全ての寄付を見る
+
+#------------------------------------------------------------------------------
+# Blog Page
+#------------------------------------------------------------------------------
+
+blog_page:
+  title: ニュース・最新情報
+  description: ESLintの最新ニュース、ケーススタディ、チュートリアル、リソースなどをご紹介します。
+  search:
+    title: 検索
+    hint: キーワードを入力するとリアルタイムに検索結果が表示されます。
+    clear: 検索の解除
+  categories:
+    title: カテゴリー一覧
+    everything: 全て
+  pagination:
+    title: ページ
+    first_page: 最初
+    next_page: 次
+    previous_page: 前
+    last_page: 最後
+    status: TOTAL_PAGE_COUNT ページ中の PAGE_NUMBER ページ目
+
+#------------------------------------------------------------------------------
+# Team Page
+#------------------------------------------------------------------------------
+
+team_page:
+  title: チーム紹介
+  description: >
+    ESLintを構築し、維持管理しているチームメンバーです
+  tsc:
+    title: 技術運営委員会
+    description: >
+      リリースを管理し、機能要求をレビューし、ESLintが適切に維持されるように定期的に
+      ミーティングを行う人たちです。
+  reviewers:
+    title: レビュアー
+    description: >
+      新機能をレビューし、実装する人々
+  committers:
+    title: コミッター
+    description: >
+      バグをレビューして修正し、問題のトリアージを支援する人々
+  alumni:
+    title: アルムナイ
+    description: >
+      以前、ESLintの保守を手伝っていた元チームメンバー
+
+#------------------------------------------------------------------------------
+# Sponsors Page
+#------------------------------------------------------------------------------
+
+sponsors_page:
+  title: スポンサー様
+  description: >
+    現在、BACKER_COUNT の企業、団体、個人が
+    ESLintの継続的な維持・発展を支援するために、毎月総額MONTHLY_DONATIONSの寄付をしています。
+  hero:
+    cta:
+      text: スポンサーになる
+      url: /donate
+  tiers:
+    platinum:
+      title: プラチナスポンサー
+      description: >
+        毎月2000ドルの寄付で、プラチナスポンサーとなり、40時間のメンテナンスと開発に貢献します。
+        2時間の専属サポートサービスをもらえます。サポートはチャットや画面共有による遠隔サポートの形でできます。
+        プラチナスポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+        また フォロワー数2万人以上のESLintのTwitterアカウントからお礼をツイートします。
+        サポートセッションをご希望の方は、ESLintチームまでご連絡ください。
+    gold:
+      title: ゴールドスポンサー
+      description: >
+        毎月1000ドルの寄付で、ゴールドスポンサーとなり、20時間のメンテナンスと開発に貢献します。
+        ゴールドスポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+        また フォロワー数2万人以上のESLintのTwitterアカウントからお礼をツイートします。
+    silver:
+      title: シルバースポンサー
+      description: >
+        毎月500ドルの寄付で、シルバースポンサーとなり、10時間のメンテナンスと開発に貢献します。
+        シルバースポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+    bronze:
+      title: ブロンズスポンサー
+      description: >
+        毎月200ドルの寄付で、ブロンズスポンサーとなり、4時間のメンテナンスと開発に貢献します。
+        ブロンズスポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+    technology:
+      title: テクノロジースポンサー
+      description: >
+        技術スポンサーはオープンソースのエコシステムと私たちの仕事への貢献の一環として、製品・サービスを無償で使わせていただいています。
+    backers:
+      title: バッカー
+      description: >
+        バッカー（支援者）は、ESLintを維持するための資金を毎月提供します。
+
+  recent_donations:
+    title: 最近の一時的な寄付
```

As I pointed at `homepage > donations > title`,
```suggestion
    title: 最近の単発寄付
```
To me, `一時的な` sounds like temporary rather than one-time.

---

If we use the word "単発寄付", I think we also need to change L379, L390, and L531-534.

(But I don't know the word "単発寄付" so I'm not sure if it's a good idea to use it. The word "単発寄付" seems to exist, but I don't know if it applies here.)

---

Yes I preferred using the word `一時的` in other places, for consistency

---

My suggestion was just an idea and I would not insist on it. Please go ahead with `一時的` 👍